binchihua/src/orch/orch.c

```diff
@@ -1,11 +1,11 @@
-#include <stdio.h>
-#include <stdlib.h>
-
 #include "../../../libchihua-msg/include/common/common.h"
 #include "../../../libchihua-msg/include/orchestrator.h"
 #include "../ini/config.h"
 #include "opt.h"
 
+#include <stdio.h>
```

Comment:
So here you have placed the system headers second, after local includes

Reply:
Oh, my mistake. Shouldn't be like that. Changed.